test/unit/specs/components/staking/TabValidators.spec.js

```diff
@@ -1,49 +1,95 @@
-import setup from "../../../helpers/vuex-setup"
+import { shallowMount } from "@vue/test-utils"
 import TabValidators from "renderer/components/staking/TabValidators"
 import lcdClientMock from "renderer/connectors/lcdClientMock.js"
 
-const { stakingParameters } = lcdClientMock.state
+const { candidates } = lcdClientMock.state
```

Comment:
do we still have to use `lcdClientMock`?

Reply:
No, good catch